robottelo/config/casts.py

```diff
@@ -70,3 +70,30 @@ class Tuple(List):
     """
     def __call__(self, value):
         return tuple(super(Tuple, self).__call__(value))
+
+
+class Dict(List):
+    """Cast a comma separated list of key=value to a dict.
+
+    :param str value: A comma separated string to cast to a dict.
+    """
+    def __call__(self, value):
+        return dict(v.split('=') for v in super(Dict, self).__call__(value))
```

Comment:
@rochacbruno you have a good point, but in this case I think the cast class should be as simple as possible and depending on the higher layer it can define if spaces will matter or not.

Let's have your suggestion in mind for future improvements if needed.

Thank you.